exporter.py

```diff
@@ -3,21 +3,33 @@
 import time
 import os
 import importlib
+import requests
+import logging
 from prometheus_client import start_http_server
 from prometheus_client.core import REGISTRY
 from optparse import OptionParser
 from tools.helper import yaml_read
 
 
 def default_collectors():
-    return [i for i in yaml_read(os.environ['CONFIG'])['default_collectors']]
+    return [collector for collector in yaml_read(os.environ['CONFIG'])['default_collectors']]
+
+
+def parse_params(logger):
+    # init logging here for setting the log level
+    formatter = logging.Formatter('[%(asctime)s] [%(levelname)s] %(message)s')
+    ConsoleHandler = logging.StreamHandler()
+    logger.addHandler(ConsoleHandler)
+    ConsoleHandler.setFormatter(formatter)
```

Comment:
Can we move this one line up? Configuring the handler and adding it afterwards is reading better